dotnet/Sample/Sample1_AnalyzeText.cs

```diff
@@ -0,0 +1,44 @@
+using Azure.AI.ContentSafety;
+
+namespace Azure.AI.ContentSafety.Dotnet.Sample
+{
+    class ContentSafetySampleAnalyzeText
+    {
+        public void AnalyzeText()
```

Comment:
Can we define the class as static and this methos as static, so that no need to new object in Program.cs?